src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,18 @@
+.stars {
+  display: flex;
+
+  &__star {
```

Comment:
Add a comment explaining the purpose of the .stars__star class, as it might not be self-explanatory to others.